### PR TITLE
Remove wordwrapping from web docs

### PR DESCRIPTION
**Pull request by danielbachhuber**
Fixes https://github.com/wp-cli/wp-cli/issues/4284

### CODE REVIEW

File: bin/command.php [hunk 1]
```diff
@@ -335,6 +335,31 @@ private static function gen_cmd_pages( $cmd, $parent = array() ) {
 			$docs = preg_replace( '/^(\[?)--/m', '\1\--', $docs );
 			$docs = preg_replace( '/^\s\s--/m', '  \1\--', $docs );
 
+			// Remove wordwrapping from docs
+			// Match words, '().,;', and --arg before/after the newline
+			$bits = explode( "\n", $docs );
+			$in_yaml_doc = false;
+			for ( $i=0; $i < count( $bits ); $i++ ) {
```
Comment (schlessera) [inline bin/command.php]: CS: spaces around assignment

File: commands/cli/alias.md [hunk 2]
```diff
@@ -3,9 +3,7 @@
 List available WP-CLI aliases.
 
 Aliases are shorthand references to WordPress installs. For instance,
```
Comment (schlessera) [inline commands/cli/alias.md]: This case does not seem to work correctly yet.

File: commands/db/check.md [hunk 3]
```diff
@@ -3,11 +3,9 @@
 Check the current status of the database.
 
 Runs `mysqlcheck` utility with `--check` using `DB_HOST`,
-`DB_NAME`, `DB_USER` and `DB_PASSWORD` database credentials
-specified in wp-config.php.
+`DB_NAME`, `DB_USER` and `DB_PASSWORD` database credentials specified in wp-config.php.
```
Comment (schlessera) [inline commands/db/check.md]: Another edge case that doesn't work yet.

File: commands/db/create.md [hunk 4]
```diff
@@ -3,8 +3,7 @@
 Create a new database.
 
 Runs `CREATE_DATABASE` SQL statement using `DB_HOST`, `DB_NAME`,
-`DB_USER` and `DB_PASSWORD` database credentials specified in
-wp-config.php.
+`DB_USER` and `DB_PASSWORD` database credentials specified in wp-config.php.
```
Comment (schlessera) [inline commands/db/create.md]: Another one.

File: commands/db/drop.md [hunk 5]
```diff
@@ -3,8 +3,7 @@
 Delete the existing database.
 
 Runs `DROP_DATABASE` SQL statement using `DB_HOST`, `DB_NAME`,
-`DB_USER` and `DB_PASSWORD` database credentials specified in
-wp-config.php.
+`DB_USER` and `DB_PASSWORD` database credentials specified in wp-config.php.
```
Comment (schlessera) [inline commands/db/drop.md]: Well, I'll be brief: all paragraphs starting with a backtick don't work.

File: bin/command.php (hunk 1 above)
Comment (danielbachhuber) [reply bin/command.php]: PHPCS isn't enforced in this repo

File: commands/db/drop.md (hunk 5 above)
Comment (danielbachhuber) [reply commands/db/drop.md]: Fixed in a7d379f

**Comment by danielbachhuber** [comment]
@schlessera Backticks are now addressed.

File: bin/command.php (hunk 1 above)
Comment (schlessera) [reply bin/command.php]: Wouldn't there be merit in being "consistently" consistent? :)

Comment (danielbachhuber) [reply bin/command.php]: Not until it's enforced through CI. CS feedback should only happen through CI. If it's happening in a pull request, then it's not an enforceable standard.

**Pull request merged**